crates/ruff/src/rules/flake8_simplify/rules/ast_if.rs

```diff
@@ -504,83 +559,50 @@ fn ternary(target_var: &Expr, body_value: &Expr, test: &Expr, orelse_value: &Exp
     node1.into()
 }
 
-/// Return `true` if the `Expr` contains a reference to `${module}.${target}`.
-fn contains_call_path(expr: &Expr, target: &[&str], semantic: &SemanticModel) -> bool {
+/// Return `true` if the `Expr` contains a reference to any of the given `${module}.${target}`.
+fn contains_call_path(expr: &Expr, semantic: &SemanticModel, targets: &[&[&str]]) -> bool {
```

Comment:
Nit: I think @charliermarsh started to move `semantic` to the back because it is only an *auxilary* argument, necessary to perform the job. The important arguments that this function is operating on are the `expr` and the `targets`.